src/app/main/component/ubs/components/ubs-order-details/ubs-order-details.component.scss

```diff
@@ -312,3 +312,86 @@ button {
 li {
   list-style: none;
 }
+
+@media (min-width: 481px) and (max-width: 767px) {
+  .label,
+  .bag-name {
+    font-size: 0.8rem;
+  }
+
+  .messages-container {
+    width: 65%;
+  }
+
+  .totalInfo {
+    span {
+      font-size: 0.8rem;
+    }
+
+    .money-saved {
+      font-size: 0.65rem;
+    }
+  }
+
+  h3 {
+    font-size: 1.3rem;
+  }
+
+  h5 {
+    font-size: 1.1rem;
+  }
+
+  button {
+    font-size: 1rem;
+  }
+
+  p,
+  a,
+  textarea,
+  small,
+  input {
+    font-size: 0.8rem;
+  }
+}
```

Comment:
it would be better to use only min width https://www.solodev.com/blog/web-design/media-queries-and-mobile-css-best-practices.stml
https://anglestudios.co.uk/blog/why-mobile-first-web-design-is-becoming-more-important/

Reply:
Yea, it's a good idea, but it's a good idea when we building CSS from 0 and when we want to use Mobile First. But in this case, we have the opposite situation. The CSS that was written here was built with Dekstop First methodology and it's a pretty old file. So it's useless now to refactor all code to Mobile First media queries.